ams/api/src/main/java/com/netease/arctic/ams/api/metrics/IcebergCommitReport.java

```diff
@@ -32,6 +32,7 @@ public class IcebergCommitReport implements MetricReport {
   public static final String SEQUENCE_NUMBER = "sequence-number";
   public static final String OPERATION = "operation";
   public static final String METADATA = "metadata";
+  public static final String COMMIE_TIME = "commit-time";
```

Comment:
Is `commit-timestamp` more appropriate as the value is the timestamp rather than a date time?

Reply:
done